projects/packages/search/.eslintrc.js

```diff
@@ -1,9 +1,11 @@
+const loadIgnorePatterns = require( '../../../tools/js-tools/load-eslint-ignore.js' );
+
 module.exports = {
 	// Use root level ESlint configuration.
 	// JavaScript files inside this folder are meant to be transpiled by Webpack.
 	root: true,
 	extends: [ '../../../.eslintrc.react.js' ],
-	ignorePatterns: [ '**/external/*.js' ],
+	ignorePatterns: loadIgnorePatterns( __dirname ),
```

Comment:
@anomiex Can you comment on this -- this came from the `master` merge. Search added the `external` ignorePattern and you added the `loadIgnorePatterns`.

Should only one remain, both?

Reply:
The `**/external/*.js` should probably be put into a `projects/packages/search/.eslintignore`